init.zsh

```diff
@@ -3,6 +3,17 @@ autoload -Uz run-help
 bindkey '^[h' run-help
 bindkey '^[H' run-help
 
+if (( ! ${+HELPDIR} )); then
+  local dir
+  for dir (/usr/local/share/zsh/help /usr/share/zsh/$ZSH_VERSION/help /usr/share/zsh/help); do
+    if [[ -d $dir ]]; then
+      typeset -g HELPDIR=$dir
```

Comment:
[Style guide](https://github.com/zimfw/zimfw/wiki/Code-Style-Guide) changes:
```suggestion
  for dir in /usr/local/share/zsh/help /usr/share/zsh/$ZSH_VERSION/help /usr/share/zsh/help; do
    if [[ -d ${dir} ]]; then
      typeset -g HELPDIR=${dir}
```

Reply:
Also, I'd rather have this whole `if (( ! ${+HELPDIR} )) ... fi` block at the beginning of the script, since this must be set before running run-help for the first time. I know we're just autoloading it, but it would be less counter-intuitive to have that block on the top.